File: dev-docs/snippets/databases/transfer-ownership.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Transfer database ownership to another user.
export async function transferOwnership(
  client: JsBaoClient,
  databaseId: string,
  newOwnerId: string,
) {
  // #region example
  const result = await client.databases.transferOwnership(databaseId, newOwnerId);
  // #endregion example
  return result;
}

File: dev-docs/snippets/databases/transfer-ownership.swift
import JsBaoClient

// Transfer database ownership to another user.
func transferOwnership(client: JsBaoClient, databaseId: String, newOwnerId: String) async throws {
  // #region example
  let result = try await client.databases.transferOwnership(
    databaseId: databaseId,
    newOwnerId: newOwnerId
  )
  // #endregion example
  _ = result
}
